apps/dashboard/src/data/ui/actions.js

```diff
@@ -53,3 +59,15 @@ export function redirect( path ) {
 		path,
 	};
 }
+
+export function fetchOngoing() {
```

Comment:
Like I said, very bad idea to have a single state branch to handle all possible requests. This belongs in the project state, where we should probably add `saving` and `loading` keys.